helix-view/src/info.rs

```diff
@@ -65,4 +65,12 @@ impl Info {
         infobox.width = 30; // copied content could be very long
         infobox
     }
+
+    pub fn from_selected_register(registers: &Registers, register: char) -> Self {
+        let title = "Collected bits";
```

Comment:
This was an arbitrary place holder I chose. Open to better ideas. "Register contents" maybe?